Hyper/Camera.cs

```diff
@@ -11,7 +11,7 @@ public class Camera : Commandable
 
         public Vector3 ReferencePointPosition { get; set; } = Vector3.Zero;
 
-        private Vector3 _front = -Vector3.UnitZ;
+        public Vector3 Front = -Vector3.UnitZ;
```

Comment:
As far as I know camera front should always be pointing at negative z direction. Exposing this as get-only seems like a better option

Reply:
done